tools/PI/DevHome.PI/Controls/EditToolsControl.xaml

```diff
@@ -27,17 +29,21 @@
             CanUserReorderColumns="True"
             CanUserResizeColumns="True"
             ColumnHeaderHeight="32"
-            MaxColumnWidth="400"
             FrozenColumnCount="0"
             GridLinesVisibility="None"
-            HeadersVisibility="Column">
+            HeadersVisibility="Column"
+            ItemsSource="{x:Bind local:ExternalToolsHelper.Instance.AllExternalTools}"
+            SelectionChanged="ToolsDataGrid_SelectionChanged">
             <controls:DataGrid.Columns>
-                <controls:DataGridTextColumn x:Uid="EditToolsNameColumn" Binding="{Binding Name}" Width="150"/>
+                <controls:DataGridTextColumn x:Uid="EditToolsNameColumn" Binding="{Binding Name}" MinWidth="150"/>
+                <controls:DataGridTextColumn x:Uid="EditToolsNameColumn" Binding="{Binding Executable}" Width="*"/>
             </controls:DataGrid.Columns>
         </controls:DataGrid>
 
         <StackPanel x:Name="GridButtonPanel" Orientation="Horizontal" Grid.Row="1" HorizontalAlignment="Right" Margin="0,4,0,8">
-            <Button x:Uid="DeleteToolButton" x:Name="DeleteToolButton" Width="120" Height="36" Margin="4,0,0,0" Click="DeleteToolButton_Click"/>
+            <Button 
+                x:Uid="UnregisterToolButton" x:Name="UnregisterToolButton" Width="120" Height="36" Margin="4,0,0,0" 
+                Click="UnregisterToolButton_Click"/>
```

Comment:
For readability, please put all parameters on one line, or all get their own line.